onnx_tf/common/handler_helper.py

```diff
@@ -0,0 +1,64 @@
+import re
+import warnings
+
+from onnx import defs
+
+from . import op_name_to_lower
+from onnx_tf.handlers.frontend import *  # noqa
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+
+
+def __get_all_subclasses(clazz, except_regex=None):
```

Comment:
Are you using `clazz` to avoid name conflict with standard keyword `class`? If so, `class_` is the right way to go according to PEP8 and I quote:
```
single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.

Tkinter.Toplevel(master, class_='ClassName')
```